src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -225,15 +225,16 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
                     const {
                       commerceCreateOrderWithArtwork: { orderOrError },
                     } = data
-                    if (orderOrError.error) {
+
+                    if (orderOrError?.error) {
                       this.onMutationError(
                         new ErrorWithMetadata(
-                          orderOrError.error.code,
-                          orderOrError.error
+                          orderOrError?.error?.code,
+                          orderOrError?.error
```

Comment:
I know this is a draft PR, but its important to not over-use the `?` operator and to understand that using it is the equivalant of saying to the next dev: "I'm not sure if this code works or not!" 

If suddenly these things are undefined, or are erroring out, then its likely a bug was introduced somewhere that we need to fix before proceeding.

This is a great blog post on this subject: https://blog.jim-nielsen.com/2020/web-technologies-and-syntax

Reply:
Also here to unblock the work I'm doing for the scope of this ticket/get the page to load. We realized there is a bug in MP where some of the CRUD addresses mutations don't have a `code` property on the error response object.

Reply:
> Also here to unblock the work I'm doing for the scope of this ticket/get the page to load

We've had this code in place for a long time without issue though. If there's suddenly a problem, perhaps a recent bug was introduced? But the solution isn't to suddenly make things like this optional in order to quickly finish a JIRA ticket, but to track down the root of the problem and solve it there.